docs/element_samples.ad

```diff
@@ -0,0 +1,133 @@
+---
+layout: default
+title: "GENIVI SOTA Project — Software, Over the Air"
```

Comment:
Oops. That was just for Timo to see the various document elements that his design needed to include; I had it stashed but must have forgotten that I brought it back. I'll remove it.